protocol/x/clob/memclob/memclob.go

```diff
@@ -1510,6 +1522,10 @@ func (m *MemClobPriceTimePriority) mustAddOrderToOrderbook(
 	}
 
 	m.openOrders.mustAddOrderToOrderbook(ctx, newOrder, forceToFrontOfLevel)
+
+	// Send an orderbook update to grpc streams.
+	orderbookUpdate := m.GetOrderbookUpdatesForOrderPlacement(ctx, newOrder)
+	m.clobKeeper.SendOrderbookUpdates(orderbookUpdate, false)
```

Comment:
i don't feel too strongly about this comment but would it make more sense to push these orderbook updates down a callstack level into the `func (m *memclobOpenOrders) mustRemoveOrder` memclobOpenOrders functions? Don't think we do this currently but in the future if we ever do direct order removal operations on the memclob open orders function orderbook updates won't be emitted.

Reply:
Same with the removal below

Reply:
thought about this when I am implementing but `memclobOpenOrders` does not hold a reference to `memclob` and therefore does not hold to reference to clob keeper to send messages 😢

Reply:
let's keep it that way for now? memclob holding a reference to clob is already kind of ugly in my opinion so don't want to introduce another backwards pointer to the parent